app/util/analytics.py

```diff
@@ -130,11 +123,14 @@ def get_actual_test_count(self):
             elif selenium_test in line:
                 self.selenium_test_count = self.selenium_test_count + 1
 
+    def __convert_to_sec(self, time):
+        seconds_per_unit = {"s": 1, "m": 60, "h": 3600, "d": 86400, "w": 604800}
+        return int(str(time)[:-1]) * seconds_per_unit[str(time)[-1]]
```

Comment:
Just int value is also possible and valid format, e.g. `duration: 300` will be the same as `duration: 300s`